SubZero/src/controller/controllers/Controller.cpp

```diff
@@ -1,18 +1,14 @@
 /*
  * Controller.cpp
  *
- *  Created on: Jan 7, 2015
- *      Author: mda
+ *  Created on: Mar 12, 2015
+ *      Author: ed
  */
 
 #include "Controller.h"
 
-Controller::Controller() {
-	// TODO Auto-generated constructor stub
+Controller::Controller(list <Model> *modelPointer){}
```

Comment:
Since the * implies the pointer, there is no need for us to repeat that it is a pointer in the name